detection_rules/schemas/registry_package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+"""Definitions for packages destined for the registry."""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Type
+
+from marshmallow import Schema, validate
+from marshmallow_dataclass import class_schema
+
+from .definitions import ConditionSemVer, SemVer
+
+
+@dataclass
+class RegistryPackageManifest:
+    """Base class for registry packages."""
+
+    conditions: Dict[str, ConditionSemVer]
+    version: SemVer
+
+    categories: List[str] = field(default_factory=lambda: ['security'])
+    description: str = 'Rules for the detection engine in the Security application.'
+    format_version: SemVer = field(metadata=dict(validate=validate.Equal('1.0.0')), default='1.0.0')
+    icons: list = field(default_factory=list)
+    internal: bool = True
+    license: str = 'basic'
+    name: str = 'detection_rules'
+    owner: Dict[str, str] = field(default_factory=lambda: dict(github='elastic/protections').copy())
```

Comment:
no need to do a copy. the lambda builds it on the fly.
you could do this though, which could be more performant but is less obvious to read. Basically you pass it the bound  `.copy` method as the callable:

```suggestion
    owner: Dict[str, str] = field(default_factory={"github": "elastic/protections"}.copy)
```

```suggestion
    owner: Dict[str, str] = field(default_factory=dict("github": "elastic/protections").copy)
```